src/App.tsx

```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
+import { useState } from 'react';
+import { GoodList } from './components/GoodList';
```

Comment:
Ensure that the `GoodList` component is converted to TypeScript as per the task requirements. Check the `GoodList` component file to confirm its type.